FirebaseAuth/Tests/Sample/ApiTests/FIRAuthApiTestsBase.h

```diff
@@ -16,9 +16,9 @@
 
 #import <XCTest/XCTest.h>
 
-#import "FIRApp.h"
+#import <FirebaseCore/FIRApp.h>
```

Comment:
Does this worth with SwiftPM? Or are these tests just not running yet?

Reply:
Yep, no tests are running with SwiftPM - so this is just a revert back to master for now.